app/views/employment/CheckEmploymentDetailsView.scala.html

```diff
@@ -118,6 +141,8 @@
         @formWithCSRF(action = controllers.employment.routes.CheckEmploymentDetailsController.submit(taxYear, employmentDetails.employmentId)) {
             @button("common.saveAndContinue", classes = Some("govuk-!-margin-top-6"))
         }
+    } else {
+        @employmentSource
```

Comment:
and can use the formHelper with the button helper here
like https://github.com/hmrc/income-tax-employment-frontend/blob/main/app/views/templates/TimeoutPage.scala.html#L33